tests/test_threaded_backend.py

```diff
@@ -60,14 +61,15 @@ def test_empty_queue(self):
         self.assertEqual(0, self.backend.fetched_items)
 
     def test_decorator(self):
-        with freeze_time('2016-08-01 00:00:00'):
+        now = datetime.now()
```

Comment:
Otherwise you get:
```
  File "/root/.pyenv/versions/3.5.1/lib/python3.5/selectors.py", line 367, in select
    fd_event_list = self._poll.poll(timeout)
OverflowError: timeout is too large
```